### PR TITLE
Adding test and test coverage

### PR DESCRIPTION
**Pull request by ugocupcic**
image

A much better overview of the code coverage is visible here: https://codecov.io/github/shadow-robot/sr-ronex?branch=F_test_coverage

You can also install a browser plugin to display the code coverage directly in github.

The current code coverage isn't good [codecov.io](http://codecov.io/github/shadow-robot/sr-ronex?branch=F_test_coverage) but we have lots of disabled tests since they use hardware.

### CODE REVIEW

File: project.rosinstall [hunk 1]
```diff
@@ -0,0 +1,9 @@
+- git:
+    local-name: ros_ethercat
+    uri: https://github.com/shadow-robot/ros_ethercat.git
+    version: indigo-devel
+
+- git:
```
Comment (ugocupcic) [inline project.rosinstall]: you are stupid

File: sr_ronex_drivers/test/test_ethercat_drivers.cpp [hunk 2]
```diff
@@ -49,7 +49,12 @@ TEST(RonexEthercatDrivers, constructor )
 
   EtherCAT_FMMU_Config fmmu(0);
   EtherCAT_PD_Config pdcfg(0);
-  EtherCAT_SlaveHandler sh(0, 0, 0, serial,EC_FixedStationAddress( (uint16_t) 0 ), &fmmu, &pdcfg, 0);
+  EtherCAT_MbxConfig mbxcfg;
+  EtherCAT_DataLinkLayer dll;
+  EC_Logic logic;
+  EtherCAT_PD_Buffer pdbuf(&logic, &dll);
+
+  EtherCAT_SlaveHandler sh(0, 0, 0, serial,EC_FixedStationAddress( (uint16_t) 0 ), &fmmu, &pdcfg, &mbxcfg, &dll, &logic, &pdbuf);
```
Comment (toliver) [inline sr_ronex_drivers/test/test_ethercat_drivers.cpp:57]: So this test was failing since we changed the signature of this function?

**Comment by toliver** [comment]
Looks good.

File: sr_ronex_drivers/test/test_ethercat_drivers.cpp (hunk 2 above)
Comment (ugocupcic) [inline sr_ronex_drivers/test/test_ethercat_drivers.cpp:57]: yes

**Pull request merged**